creator-node/src/index.ts

```diff
@@ -116,7 +117,13 @@ const startApp = async () => {
 
   // Some Services cannot start until server is up. Start them now
   // No need to await on this as this process can take a while and can run in the background
-  serviceRegistry.initServicesThatRequireServer()
+  serviceRegistry.initServicesThatRequireServer().then((_: any) => {
```

Comment:
why use a `.then` instead of an `await`?
also i think we can safely just call this function inside `serviceRegistry` instead of passing it in here?

Reply:
I was doing this to not wait for this, but .then waits anyway so that was pointless. I moved it into serviceRegistry at the end of initting services so they'll all be initialized when BullBoard references them